src/components/PopoverContainer/PopoverContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './PopoverContainer.module.css';
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  children: React.ReactNode;
+}
+
+// Default settings for any popover container using PopperJS
+export const defaultPopoverModifiers = [
+  {
+    name: 'offset',
+    options: {
+      offset: [0, 10], // spaces the popover from the trigger element
```

Comment:
I think the mocks have an offset of 8px

Reply:
This data is a little weird right now. I am not using these values in this PR, but moved this from another file, so all popover containers would have their settings in one place. This does reveal a discrepancy between popovers tho... once they all implement popperJS, they should all use **this** config (this will also prevent circular dependencies) and we would change the values to 8 if that's correct for everyone.

Reply:
Makes sense, thanks